polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -65,8 +66,12 @@ export interface TooltipProps {
   onOpen?(): void;
   /* Callback fired when the tooltip is dismissed */
   onClose?(): void;
+  /** Used for Tooltips displaying information regarding a keyboard shortcut */
+  keyboardShortcut?: string;
```

Comment:
Should this prop be generic? The text passed in doesn't tie into any additional logic, so I'm not sure about tying the prop to the concept of keyboard shortcuts.

```suggestion
  /** Subdued text rendered after the tooltip content */
  suffix?: string;
```